djangocms_versioning/cms_config.py

```diff
@@ -143,19 +143,25 @@ def label_from_instance(obj, language):
     return "{title} ({path})".format(title=obj.get_title(language), path=obj.get_path(language))
 
 
-def emit_page_change(version):
-    # Trigger a post object operation
-    post_obj_operation.send(
-        sender=version.__class__,
-        operation=CHANGE_PAGE_TRANSLATION,
-        request=None,
-        token=None,
-        obj=version.content,
-    )
+def emit_page_content_change(version):
+    """
+    Sends a page content change signal for djangocms-internalsearch
+    if installed. It is used for re-indexing version state info
+    """
+    try:
+        from djangocms_internalsearch.signals import page_content_change_signal
+    except ImportError:
+        page_content_change_signal = None
+
+    if page_content_change_signal:
+        page_content_change_signal.send(
```

Comment:
Shouldn't it be a signal that handles all content objects?

Reply:
True